flink-streaming-java/src/main/java/org/apache/flink/streaming/api/environment/LocalStreamEnvironment.java

```diff
@@ -68,19 +66,4 @@ private static Configuration validateAndGetConfiguration(final Configuration con
 		effectiveConfiguration.set(DeploymentOptions.ATTACHED, true);
 		return effectiveConfiguration;
 	}
-
-	/**
-	 * Executes the JobGraph of the on a mini cluster of ClusterUtil with a user
-	 * specified name.
-	 *
-	 * @return The result of the job execution, containing elapsed time and accumulators.
-	 */
-	@Override
-	public JobExecutionResult execute(StreamGraph streamGraph) throws Exception {
-		try {
-			return super.execute(streamGraph);
-		} finally {
-			transformations.clear();
-		}
-	}
```

Comment:
I think the removal of this method makes the japicmp plugin to fail: https://travis-ci.org/apache/flink/jobs/624197613

```
17:06:02.660 [ERROR] Failed to execute goal com.github.siom79.japicmp:japicmp-maven-plugin:0.11.0:cmp (default) on project flink-streaming-java_2.11: Breaking the build because there is at least one incompatibility: org.apache.flink.streaming.api.environment.LocalStreamEnvironment.execute(org.apache.flink.streaming.api.graph.StreamGraph):METHOD_REMOVED -> [Help 1]
```

Reply:
I think the problem is that this method has not been annotated with `@Internal`